src/getProjectId.js

```diff
@@ -21,10 +22,19 @@ module.exports = function(options, allowNull = false) {
 
     if (aliasCount === 0) {
       throw new FirebaseError(
-        "No project active. Run with " +
-          clc.bold("--project <projectId>") +
-          " or define an alias by\nrunning " +
-          clc.bold("firebase use --add"),
+        "No currently active project.\n" +
+          "To run this command, you have two options:\n" +
```

Comment:
`To run this command,...` doesn't feel right. That isn't the problem being solved

`To select a project, either:` would reflect "hey, you need to select a project", then the thought continues below with "Run this command with a thing, or set an active project".

Reply:
IMO, the dev not being able to run the command is the _actual_ problem.  They don't care about "selecting a project", they care about how they can get the command to run.

Maybe we could say:
"To run this command, you need to specify a project. You have two options:

Reply:
Going with Rachel's language here, since I think it addresses the question the user is probably asking ("why can't I run this command?") as well as clarifying what you actually are doing ("specifying a project")

Reply:
The updated file (in Files changed tab) LGTM!